LondonBritishLibrary/orient/BLorient786.xml

```diff
@@ -0,0 +1,239 @@
+<incipit><locus target="#109v" xml:lang="gez"></locus>በእንተ፡ ሞቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ጊዜ፡ ፱ሰዓት፡ በዲበ፡ ዕፀ፡ መስቀል፡
+ወበእንተ፡ ፈያታዊ፡ ዘየማን፡ ወበእንተ፡ ሞት፡ ወሰይጣን፡ ዘከመ፡ ተበሃሉ፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.13">
+<locus from="125r"/>
+<title type="complete" ref="LIT2109OntheA">On the ascension of our Lord</title>
+<textLang mainLang="gez" xml:lang="gez"></textLang>
+<incipit><locus target="#125r"></locus>ድርሳን፡ በእንተ፡ ዘከመ፡ ዓርገ፡ እግዚእነ፡ ውስተ፡ ሰማያት፡ መልዕልተ፡ ኵሉ፡ ስም፡ ወነበረ፡ በየማነ፡ ኃይል፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.14">
+<locus from="132v"/>
+<title type="complete" ref="LIT2137OntheP">On the Trinity in unity</title>
+<textLang mainLang="gez"></textLang>
+<incipit><locus target="#132v" xml:lang="gez"></locus>ርድአኒ፡ ወልደ፡ እግዚአብሔር፡ ከመ፡ እትናገር፡ በእንተ፡ ህሉና፡ መስልሥትከ፡ በተዋህዶትከ፡ </incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.15">
+<locus from="148v"/>
+<title type="complete" ref="LIT6047On12ApostlesRetuaH">On the twelve apostles</title>
+<textLang mainLang="gez">በእንተ፡ ፲ወ፪ሐዋርያት፡ ዘይትነበብ፡ አመ፡ ፭ሐምሌ</textLang>
+<incipit><locus target="#148v" xml:lang="gez"></locus>ርድአኒ፡ ወልደ፡ እግዚአብሔር፡ ከመ፡ እትናገር፡ በእንተ፡ ህሉና፡ መስልሥትከ፡ በተዋህዶትከ፡ </incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.16">
+<locus from="154r"/>
+<title type="complete" ref="LIT2145OntheT">On the transfiguration of our Lord</title>
+<textLang mainLang="gez"></textLang>
+<incipit><locus target="#154r" xml:lang="gez"></locus>ድርሳን፡ ዘደብረ፡ ታቦር፡ ዘከመ፡ ተወለጠ፡ ራዕዩ፡ ለእግዚእነ፡ በውስተ፡ ደብረ፡ ታቦር፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.17">
+<locus from="160v"/>
+<title type="complete" ref="LIT2124OntheF">On the four beasts</title>
+<textLang mainLang="gez"></textLang>
+<incipit><locus target="#160v" xml:lang="gez"></locus>ድርሳን፡ ዘ፬እንስሳ፡ ክቡራን፡</incipit>
+</msItem>
+
+<msItem xml:id="ms_i2">
+<colophon xml:id="coloph1" xml:lang="gez">
+        <locus from="167v"/>
+        ዝመጽሐፍ፡ ዘእራስ፡ <persName role="owner" ref="PRS2021IHA">ሣህለ፡ ሥላሴ፡</persName>
+         ወአቡሁ፡ <persName ref="PRS10076WasanS">ወልደ፡ ጊዮርጊስ፡ ወሰን፡ ሰገድ፡</persName>
+        ወእሙ፡ <persName ref="PRS10604Zannaba">ኢያሲሜር፡ ዘነመ፡ ወርቅ፡</persName> ወጸሐፊሁ፡
+        ኃጥእ፡ ወድኩም፡ <persName role="scribe" ref="PRS13030AsbaGiyorgis">አስበ፡ ጊዮርጊስ</persName>።
+            </colophon>
+</msItem>
+</msContents>
+<physDesc>
+
+  <objectDesc form="Codex">
+      <supportDesc>
+          <support>
+              <material key="parchment"/>
+          </support>
+          <extent>
+              <measure unit="leaf">168</measure>
+              <measure unit="blank">2</measure>
+<dimensions type="outer" unit="in">
+<height>14.03</height>
+<width>10.25</width>
+</dimensions>
+
+            </extent></supportDesc>
+
+            <layoutDesc>
+                <layout columns="3" writtenLines="25">
+                </layout>
+            </layoutDesc>
+
+</objectDesc>
+
+<handDesc>
+
+    <handNote script="Ethiopic" xml:id="h1">
+        <desc>Written in a good hand, during the earlier part of the 19th Century (before <date when-custom="1812"></date>)
+
+        </desc>
+
```

Comment:
notBefore 1800 notAfter 1812. The information on the dating should be inside th <date>

Reply:
(before `<date when-custom="1812"></date>`) is not necessary here, or you should encode it with notBefore